opportunity/README.md

```diff
@@ -73,12 +86,69 @@ resources below.
 
 #### Link Object
 
-Each link in the links array must be a [Link](https://github.com/radiantearth/stac-spec/blob/master/commons/links.md#link-object) Object.
+Each link in the links array must be a
+[Link](https://github.com/radiantearth/stac-spec/blob/master/commons/links.md#link-object)
+Object.
 
 ##### rel=create-order
 
-This Link object fully describes the necessary HTTP request to submit an Order for this Opportunity via [Create Order](https://github.com/stapi-spec/stapi-spec/tree/main/order#create-order-request).
+This Link object fully describes the necessary HTTP request to submit an Order
+for this Opportunity via
+[Create Order](https://github.com/stapi-spec/stapi-spec/tree/main/order#create-order-request).
 
 To conform to the Create Order spec, use `"method": "POST"`.
 
 If no Body parameters apply to an Opportunity, use `"body": {}`.
+
+## Async Opportunity Search
+
+- **Conformance URI:** <https://stapi.example.com/v0.1.0/async-opportunities>
+
+STAPI has an optional conformance class providing support for async opportunity
+searches, to accommodate searches for products that require more time than is
+allowable by the duration of an HTTP request. This support requires persisting
+the search state and matching opportunities outside the context of the initial
+request, so they are available for retrival at a later time by the requestor.
+The Opportunity Search Record is an additional entity defined to model the
+required search state.
+
+### Opportunity Search Record
+
+Returned by an async opportunity search. Can also be retrieved directly.
+
+| Field Name | Type                                                                       | Description |
+|------------| -------------------------------------------------------------------------- | ----------- |
+| id         | string                                                                     | **REQUIRED.** Opportunity search record ID. |
+| productId  | string                                                                     | **REQUIRED.** Product identifier. This should be a reference to the [Product](https://github.com/Element84/stapi-spec/blob/main/product/README.md) being searched. |
+| status     | SearchStatus                                                               | **REQUIRED.** The current search status. |
+| links      | \[[Link Object](#link-object)]                                             | List of link objects to resources and related URLs. There **must** be a `rel=opportunities` link to the corresponding opportunity collection when the search is completed. See [Link Object - rel=opportunities](#rel=opportunities) below. |
+
+#### rel=opportunities
+
+This Link object provides the means of retireving the search results when the
+search is completed. That is, it should include the equivalent of `GET
+/products/{productId}/opportunities/{collectionId}` where `productId` is the
```

Comment:
maybe rename `collectionId` to `opportunitiesCollectionId` -- whenever I see "collection", I always think of a STAC Collection

Reply:
I just didn't want to make an overly-long table line even longer, but you are right, it should be `opportunitiesCollectionId`.